dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -230,6 +236,32 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
         return tool;
     }
 
+    private static List<ImageData> processImageDataForWorkflowVersions(final Version version, final ToolVersion toolVersion) {
+        toolVersion.setIsProduction(true);
+        Set<Image> images = version.getImages();
+        List<ImageData> trsImages = new ArrayList<>();
+
+        for (Image image : images) {
+            ImageData imageData = new ImageData();
+            imageData.setImageType(ImageType.DOCKER);
+            //TODO: For workflows, it's just quay for now, but that'll change. Should add column in image table to store registry.
+            imageData.setRegistryHost("quay.io");
```

Comment:
I know it's for now, but this sort of scares me if we get distracted and don't come back to it. Is there a ticket for this?

Reply:
It was something I realized while doing this PR, so just made an issue here https://github.com/dockstore/dockstore/issues/3275